pageStyles/candidates.module.scss

```diff
@@ -0,0 +1,43 @@
+﻿@import 'pageStyles/constants';
+
+.link {
+  margin: unset;
+}
+
+.desktopButton {
+  background-color: $softwireYellow;
+  border: none;
+  padding: 5px 15px;
+  font-size: 20px;
+  font-weight: bold;
+  margin-top: 10px;
+  position: fixed;
```

Comment:
This looks a little suspicious - do we definitely want it to be `fixed`? From the designs I'd have thought we'd want this to scroll with the rest of the page?

Reply:
Now not fixed